.github/workflows/ci.yml

```diff
@@ -227,9 +227,32 @@ jobs:
       - name: Rename Binary with Target Name
         run: mv target/${{ matrix.target }}/release/tailcall target/${{ matrix.target }}/release/tailcall-${{ matrix.target }}
 
+      # Copy the CLI binary and rename it to include the name of the target platform
+      - name: Copy CLI binary
+        id: copy-cli
+        run: |
+          mkdir dist
+          cp target/${{ matrix.target }}/release/tailcall ./dist/tailcall-${{ matrix.target }}
+
       - name: Upload ${{ matrix.target }} Binary
         uses: xresloader/upload-to-github-release@v1
         with:
           release_id: ${{ needs.draft_release.outputs.create_release_id }}
           file: target/${{ matrix.target }}/release/tailcall-${{ matrix.target }}
           overwrite: true
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Generate npm packages
+        run: node npm/tailcall/scripts/generate-packages.js
```

Comment:
Let's use `npm run gen` command instead over here.

Reply:
Can you check now?